packages/components/button/package.json

```diff
@@ -1,15 +1,22 @@
 {
   "name": "@ds-name/button",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "Button component is used to trigger an action or event, such as submitting a form, opening a Dialog, canceling an action, or performing a delete operation.",
   "type": "module",
   "main": "dist/cjs/index.js",
+  "types": "./dist/main.d.ts",
```

Comment:
```suggestion
  "types": "./dist/[cjs | esm]main.d.ts",
```

Reply:
Maybe this can help: https://antfu.me/posts/publish-esm-and-cjs#tsup